src/eastwood/linters/implicit_dependencies.clj

```diff
@@ -0,0 +1,31 @@
+(ns eastwood.linters.implicit-dependencies
+  (:require [eastwood.copieddeps.dep1.clojure.tools.analyzer.ast :as ast]
+            [eastwood.copieddeps.dep9.clojure.tools.namespace.parse :as ns-parse]
+            [eastwood.util :as util]))
+
+
+(defn var->ns-symbol [var]
+  (let [^clojure.lang.Namespace ns (-> var meta :ns)]
+    (.-name ns)))
+
+
+(defn implicit-dependencies [{:keys [asts forms] :as x} opt]
```

Comment:
Please rename `opt` to `_` since it's not used.

Reply:
You got it.